test/bundle.c

```diff
@@ -110,16 +110,14 @@ static void test_check_empty_bundle(BundleFixture *fixture,
 	g_assert_false(res);
 	g_assert_error(ierror, G_IO_ERROR, G_IO_ERROR_PARTIAL_INPUT);
 	g_assert_null(bundle);
```

Comment:
If bundle is supposed to be NULL here, why have a g_autoptr for it?_

Reply:
I'd say for consistency

Reply:
Yes, maybe someone will copy this test and change it.

Reply:
Ok, convinced.